docs/learn/expert-guides/run-graph-node.md

```diff
@@ -0,0 +1,110 @@
+---
+sidebar_label: '🌐 Setting Up a Graph Node on LUKSO'
+description: 'A step-by-step guide to setting up a Graph Node on LUKSO using Docker.'
+sidebar_position: 12
+---
+
+# Run Graph Node on LUKSO
+
+There are two ways to launch a Graph Node: you can use Docker to run related images, or you can compile the [Graph Node source code](https://github.com/graphprotocol/graph-node). The steps described in this document will only be run through docker.
+
+> The instructions provided have been verified on MacOS systems. Adjustments may be needed for other operating systems.
+> The content of this guide is informational, based on third-party inputs. It's important to verify any information with the official Graph repositories (https://github.com/graphprotocol/graph-node/tree/master/docker).
+
+## Prerequisites
+
+Before diving into setting up a Graph Node, you need to have the following installed on your system:
+
+- [Docker](https://docs.docker.com/get-docker/)
+- [Docker Compose](https://docs.docker.com/compose/install/)
+- [JQ](https://jqlang.github.io/jq/download/)
+
+In this guide, you will learn how to run a Graph node against LUKSO. This guide can also be adapted for LUKSO Testnet.
+
+## Prerequisites
+
+The first step is to clone the [Graph Node repository](https://github.com/graphprotocol/graph-node/):
+
+```bash
+git clone https://github.com/graphprotocol/graph-node/ \
+&& cd graph-node/docker
+```
+
+Next, execute the `setup.sh` file. This will pull all the necessary Docker images and write the necessary information in the `docker-compose.yml` file.
+
+```bash
+./setup.sh
+```
+
+Once everything is set up, you need to modify the "Ethereum environment" inside the `docker-compose.yml` file, so that it points to the endpoint of the node you are running this Graph Node against. Note that the `setup.sh` file detects the `Host IP` and writes its value, so you'll need to modify it accordingly.
```

Comment:
Maybe you can add a link to network page to let the user chose other RPC providers:

- https://docs.lukso.tech/networks/mainnet/parameters